backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/work/impl/ElasticsearchStubWorkFactory.java

```diff
@@ -103,7 +103,9 @@ public ElasticsearchStubWorkFactory(GsonProvider gsonProvider) {
 	}
 
 	@Override
-	public ElasticsearchWork<?> delete(URLEncodedString indexName, URLEncodedString typeName) {
+	public ElasticsearchWork<?> delete(URLEncodedString indexName, URLEncodedString typeName, String tenantId) {
```

Comment:
Adding a comment so that we remember: this cannot be merged as is.

1. The implementation should change: a simple "delete" just won't work, in particular because the index, its schema and its settings must still be there after the purge (they may have been modified by the user, so we can't just restore them ourselves). A delete-by-query would be more appropriate.
2. As a consequence, The method should be named "purge" or "delete by query".
3. We should have tests for a purge in a multi-tenant environment, to be sure no data is lost.